python/ray/autoscaler/aws/tests/aws_config.yaml

```diff
@@ -1,4 +1,4 @@
-base_image: {{ env["RAY_IMAGE_NIGHTLY_CPU"] | default("anyscale/ray:nightly-py37") }}
+base_image: {{ env["RAY_IMAGE_NIGHTLY_CPU"] | default("anyscale/ray:nightly-py38") }}
```

Comment:
no idea why these were using py37...

@rickyyx ?

Reply:
I don't think there is a particular reason.